about.html

```diff
@@ -0,0 +1,307 @@
+                        <div class="col-12">
+                            <p class="about_paraghraph">
+                                <span class="about_span">Catenx Nigeria. limited. present</span> , The role of agri-entrepreneurship and farming innovations for youth in agriculture engagement in West Africa
+                            </p>
+                        </div>
+                    </div>
+                    <div class="row">
+
+                        <div class="col-12 about_paragraph_second">
+
+                            <p>
+                            <p> In particular, this year's summit is our third since incession</p>
+                            On January 20, 2016 a seminar was organized on the topic of entrepreneurship and innovations in developing countries for young peoples’ involvement in farming. Participants were mid-career professionals from Africa and Asia as well as Netherlands-based professionals that were representing knowledge, policy and private institutes. Three presenters brought in ideas from the fields of information and communication technology (ICT), Slow Food, and cooperatives for young professionals. These ideas were further explored by Catenx Nigeria limited and now we extend our materials and knowledge to growing West Africa Youths
+
+                            </p>
+                            <p><img src="./images/images/about_icon.png" alt="icon"></p>
+                            <p class="about_paragraph_box">April 13th - 14th, 2023 (Thu - Fri) <br>
+                                See you at the Catenx Summit!</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-12 ">
+                            <p class="about_buttom_paragraph">Other questions about West Africa Youth Agric-Conference? <br>
+                                <span class="organisation"><a href="#">okhaishiestevenix@yahoo.com</a> </span>
+                            </p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </div>
+        <div>
+            <!-- ---------------------Middle section about page  -->
+            <div class="container-fluid about_middle_section">
+                <div class="row">
+                    <div class="col-4"></div>
+                    <div class="col-8">
+                        <div class="about_middle_container">
+                            <h2>Catenx Nigeria limited Logo</h2>
+                            <div class="row">
+                                <div class="col-2"></div>
+                                <div class="col-10">
+                                    <div class="about_divider"></div>
+                                </div>
+                            </div>
+                        </div>
+
+
+                    </div>
+                </div>
+                <div class="row">
```

Comment:
- [ ] I noticed that this section is different from the original design, your text font size, weight and the position of each element are a bit different. Please make sure to fix the original design as much as possible :+1: 

##### Orignal design
![Screenshot_20230113_121401(1)](https://user-images.githubusercontent.com/6615394/212318567-2580c849-82a4-496a-8f61-0018523e6636.jpg)

##### Your design
![Screenshot_20230113_121753](https://user-images.githubusercontent.com/6615394/212318793-a393c68b-21ea-466e-b641-83f7114f90ed.jpg)